subsys/bluetooth/mesh/lightness_srv.c

```diff
@@ -851,6 +866,17 @@ static int bt_mesh_lightness_srv_init(struct bt_mesh_model *model)
 
 #if CONFIG_BT_SETTINGS
 	k_work_init_delayable(&srv->store_timer, store_timeout);
+
+#if IS_ENABLED(CONFIG_EMDS)
```

Comment:
No, it can not be converted to "if (IS_ENABLED(CONFIG_EMDS)) {" because the struct below is defined in emds.h and this is not included when CONFIG_EMDS is not added.